Ошибка: HTTPConnectionPool(host='s', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x00000161C0133790>: Failed to resolve 's' ([Errno 11001] getaddrinfo failed)"))
URL: http://s